Xamarin.Forms.Core/MultiBinding.cs

```diff
@@ -13,12 +10,12 @@ public sealed class MultiBinding : BindingBase
 	{
 		IMultiValueConverter _converter;
 		object _converterParameter;
-		BindingExpression _expression;
 		IList<BindingBase> _bindings;
 		BindableProperty _targetProperty;
-		bool _isApplying;
-		bool _isCreating;
-		bool _hasSuccessfullyConverted;
+		BindableObject _targetObject;
```

Comment:
this could be a WeakRef in case you hold a ref on the Binding, and not on the target